draft-zip.md

```diff
@@ -0,0 +1,114 @@
+2. **Ecosystem Benefits of Longer Time Horizons**: A reliable and long-term functioning Zcash blockchain allows users to make secure long-term plans, leading to a sustainable and virtuous adoption cycle, rather than being influenced by short-term trends.
+3. **Incentivizing Transaction Inclusion**: By providing a 40% share of transaction fees to miners, this ZIP maintains incentives for miners to prioritize including transactions in their blocks. This helps ensure the efficient processing of transactions and supports a robust and responsive network.
+4. **Future-Proofing the Network**: Diverting transaction fees into the ZSF is a forward-looking approach that prepares the Zcash network for future challenges and opportunities. It establishes a financial buffer that can be instrumental in addressing unforeseen issues and seizing strategic advantages as the Zcash ecosystem evolves.
+
+# Specification
+
+This ZIP only proposes a single modification to the transaction fees:
+1. Keep the current destination of 40% of the fees untouched, but route 60% of the fees back to the ZSF
+
+## Transaction fee routing requirements
+
+- For each transaction, 60% of the total fee are be paid to the ZSF
+- Any fractions are rounded in favour of the miner
+
+### Consensus Rule Changes
+
+The coinbase transaction at block height `height` MUST have a `zsfDeposit` that is greater than or equal to `floor((TransactionFees(height) * 6) / 10)`, where `TransactionFees(height)` is the sum of the the remaining value in the transparent transaction value pool of the non-coinbase transactions in the block at `height`.
+
+TODO ZIP Editors:
+- work out how to deal with pre-v6 transactions which don't have the zsfDeposit() field. For example, by requiring the remaining value in the transparent transaction value pool of a coinbase transaction to be greater than or equal to 60% of the fee
+- consider imposing this requirement on v6 transactions instead of an explicit `ZSF_DEPOSIT` field requirement
+- work out the necessary protocol spec changes.
+
+## Deployment
+
+The implementation of this ZIP MUST be deployed at the same time or after the Zcash Sustainability Fund is established.
+
+# Rationale
+
+We believe that is ultimately a very minor change to the protocol, and quite simple in terms of implementation overhead. Additionally – and at the time of this writing – transaction fees are so small that 60% will likely not have a major impact.
+
+If transaction fees were to increase, future ZIPs can be written to change the 60%/40% split. Finding the optimal fee split may require an iterative approach involving adjustments based on real-world data and network dynamics.
+
+In the future, other ZIPs may be written to fund the ZSF in various ways, including but not limited to:
+- ZSA fees
+- dApp-specific fees and donations
```

Comment:
Is "dApp" sufficiently well known to not require a definition or expansion?

Reply:
I've expanded it to 'decentralized applications'.